README.md

```diff
@@ -89,6 +90,22 @@ cd my-blog
 gem install
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228943487-09c1e7f9-8a1c-4d71-9b70-11f5f733e356.png)

**[OPTIONAL]**

- [ ] Kindly replace SQLite with PostgreSQL since you are using PostgreSQL in your application :+1: